src/NativeShell/src/NativeShell.java

```diff
@@ -43,7 +43,7 @@ public void run() {
                 // theme the system bars so that all models use the same color
                 // several manufacturers style the navigation bar differently
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                    cordova.getActivity().getWindow().setStatusBarColor(cordova.getActivity().getResources().getColor(android.R.color.black));
+                    cordova.getActivity().getWindow().setStatusBarColor(cordova.getActivity().getResources().getColor(android.R.color.holo_orange_dark));
```

Comment:
I'll need to revert this, was just trying to make it clear that I deployed the correct version of the app.